### DISCUSSION TITLE
Remove timer after last set is completed

### DISCUSSION THREAD

**Discussion by Kjeserud** [astashov/liftosaur#156] [Ideas]
When you complete the last set in a workout, a timer starts as normal. IMO there's no need for a timer after the last set of a workout has been completed, and in the UI it's in the way of the button to end the workout.

**Comment by yrinkel** [comment]
Yes I concur to that. No timer is needed. Also, some apps bring a pop up to allow to directly click “Finish the workout” after the last available set has been completed. 

 It’s a minor thing but will make the app feel more polished.

**Comment by galantra** [comment]
People might want to quick add a set or exercise, for example because they still have some time to kill. In my opinion, the timer should stay.

**Reply by yrinkel** [reply]
The idea is to disable the timer after the last active set is completed. 
So you add a set or exercise, the timer will still be active.

**Comment by 0xblur** [comment]
Just hit the trashcan button after your "last set". People's "last set" will differ from person to person, so I don't this is a good idea.

**Comment by tasdflkjweio** [comment]
tbh, it should be easily configurable rather than a global rule
